src/main/kotlin/entity/ChattORESpec.kt

```diff
@@ -2,6 +2,8 @@ package chattore.entity
 
 import com.uchuhimo.konf.ConfigSpec
 import chattore.pridePresets
```

Comment:
These are referred to as `presets` here but `colors` later.

Reply:
yea I took that part from the presets.kt for the config

val pridePresets = prideColors.toGradientMinimessage()
val birdflopPresets = birdflopColors.toGradientMinimessage()
val playerPresets = playerColors.toGradientMinimessage()

i did prideColors instead of pridePresets now